### DISCUSSION TITLE
how to enable service  when start app

### DISCUSSION THREAD

**Discussion by Layty** [rustdesk/rustdesk#5478] [General]
how to enable the service when start app?

**Comment by dinger1986** [comment]
On what os?

**Reply by Layty** [reply]
Android

**Comment by xlionjuan** [comment]
![Screenshot_20230901-173354](https://github.com/rustdesk/rustdesk/assets/25243894/5b260f3f-0ea1-4082-90d3-0c826f30fed9)
